docs/SBOM-AND-SOURCES.md

```diff
@@ -0,0 +1,120 @@
+# SBoM and Sources
+
+eve-os comes with a [software bill of materials (SBoM)](https://en.wikipedia.org/wiki/Software_supply_chain)
```

Comment:
Don't we capitalize this as EVE-OS elsewhere?

Reply:
I lost track. For a while we were doing it without, then with, or maybe that was just my impression.

I updated all references in this document to be EVE-OS.